api/organisations/tasks.py

```diff
@@ -290,6 +290,17 @@ def restrict_use_due_to_api_limit_grace_period_over() -> None:
         if not stop_serving and not block_access:
             continue
 
+        if not organisation.has_subscription_information_cache():
+            continue
+
+        subscription_cache = organisation.subscription_information_cache
+        api_usage = get_current_api_usage(organisation.id, "30d")
```

Comment:
Do you think `"30d"` should be a constant?

Reply:
In an upcoming branch the calls that say "30d" or similar will be replaced with datetimes so I think we can just leave it for now.